singleheader/CMakeLists.txt

```diff
@@ -22,15 +22,16 @@ find_program(BASH bash)
 
 # Under Windows, exectuting a bash script works, except that you cannot generally
 # do bash C:/path to my script. You need a  "mounted" path: /mnt/c/path
+find_package(Python3 COMPONENTS Interpreter)
 
-if (BASH AND (NOT WIN32) AND SIMDJSON_BASH)
+if (Python3 AND (NOT WIN32))
```

Comment:
There is no such variable exported by the python3 find module.

Should be [`Python3_Interpreter_FOUND`](https://cmake.org/cmake/help/latest/module/FindPython3.html#result-variables).

Reply:
Thanks, fixed! 
I do not dare to touch the windows thing at this point.